config/manual_outliers_schema.toml

```diff
@@ -0,0 +1,29 @@
+[reference]
+Description = "nan"
+Deduced_Data_Type = "Int64"
+Nullable = false
+Current_Data_Type = "str"
+Length = ">=1"
+Min_values = 11001603625
+Max_values = 19891309165
+Possible_Categorical_Values = ["nan"]
+
+[instance]
+Description = "nan"
+Deduced_Data_Type = "Int64"
+Nullable = false
+Current_Data_Type = "str"
+Length = "nan"
+Min_values = "nan"
+Max_values = "nan"
+Possible_Categorical_Values = ["nan"]
+
+[manual_outlier]
+Description = "nan"
+Deduced_Data_Type = "bool"
```

Comment:
The "bool" data-type converts all strings (including empty cells) as True. This gives us 15000 manual outliers which is what alerted us. I got this working using "boolean" instead because it's nullable. 

This is a similar problem to "str" vs "string" datatypes.

Reply:
Good spot, have updated this now.